src/ig/IGExporter.ts

```diff
@@ -1294,6 +1317,148 @@ export class IGExporter {
+        // Assign Ig.definition.resource.isExample if provided
+        if (configEntry != null && configEntry.isExample != null) {
+          resource.isExample = configEntry.isExample;
+        }
+      });
+
+      // Update IG.definition.page.name
+      this.updatePageName(this.ig.definition.page);
+
+      // Add new IG.definition.page.source[x] property
+      this.ig.definition.page.page.forEach(page => {
+        // this.ig.definition.page is the toc.html page we create
+        // All configured pages are at the next level, so start at that level
+        this.addPageSource(page, this.config.pages);
+      });
+      // Default IG.definition.page.source[x] on every page if not set
+      this.defaultPageSourceUrl(this.ig.definition.page);
+
+      // Update IG.definition.parameter
+      this.ig.definition.parameter.forEach(parameter => {
+        const guideParameterCodes: string[] =
+          this.fhirDefs
+            .fishForFHIR('http://hl7.org/fhir/guide-parameter-code', Type.CodeSystem)
+            ?.concept.map((concept: CodeSystemConcept) => concept.code) ?? [];
+
+        const code = parameter.code as string;
+        parameter.code = { code };
+        const parsedCode = parseCodeLexeme(code);
+
+        if (parsedCode.code && parsedCode.system) {
+          // If we can parse the code and we have a system and a code provided, we should use that.
+          parameter.code.code = parsedCode.code;
+          parameter.code.system = parsedCode.system;
+        } else if (guideParameterCodes.some(c => c === code)) {
+          // Otherwise, only a code was provided, so we check if it is in the bound VS
+          parameter.code.system = 'http://hl7.org/fhir/guide-parameter-code';
+        } else {
+          // If the code is not in the VS in the R5 IG resource, we default the system
+          // based on https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/Unknown.20FHIRVersion.20code.20'5.2E0.2E0-ballot'/near/298697304
+          parameter.code.system = 'http://hl7.org/fhir/tools/CodeSystem/ig-parameters';
+        }
+      });
+
+      // Add new copyrightLabel property
+      if (this.config.copyrightLabel) {
+        this.ig.copyrightLabel = this.config.copyrightLabel;
+      }
+
+      // Add new versionAlgorithm property
+      if (this.config.versionAlgorithmString) {
+        this.ig.versionAlgorithmString = this.config.versionAlgorithmString;
+      } else if (this.config.versionAlgorithmCoding) {
+        this.ig.versionAlgorithmCoding = this.config.versionAlgorithmCoding;
+      }
+
+      // Add new dependsOn.reason property
+      this.ig.dependsOn?.forEach(dependency => {
+        const configDependency = this.config.dependencies.find(
+          d => d.packageId === dependency.packageId
+        );
+        if (configDependency.reason) {
+          dependency.reason = configDependency.reason;
+        }
+      });
+    }
+  }
+
+  updatePageName(page: ImplementationGuideDefinitionPage): void {
```

Comment:
Should this be named `updatePageNameForR5` for clarity (since you would never want to do this on an R4 IG)?

Reply:
Oh yeah I like that. Do you think it's worth also adding a `ForR5` to the other two function (`addPageSource` and `defaultPageSourceUrl`) or do you think that it's implied since `source[x]` doesn't exist in R5?

Reply:
I thought about that. Since `source[x]` exists only in R5, it's probably not as important -- but it certainly wouldn't hurt!  I'll leave it up to you.

Reply:
I'll do it. In the name of consistency.